core/tests/protractor_utils/editor.js

```diff
@@ -1316,6 +1316,24 @@ var revertToVersion = function(version) {
   });
 };
 
+var readFeedbackMessages = function() {
```

Comment:
See the other functions in this file -- don't you need something along the lines of _runFromFeedbackTab? When writing infrastructure-level stuff, think about the other callers who might use this function in the future.